JeffShepherd/niche-audience
# Cypress testing

## What does this PR do?

- Install cypress library
- Add tests for home page, searching/favoriting, and sad paths

## How should it be tested in the UI?

- Run app in one terminal tab
- In another terminal tab, run ```npm run cypress:open```
- Ensure all tests pass
